examples/react-router/app/resources/lists/index.js

```diff
@@ -0,0 +1,4 @@
+export ListStore from './store'
+export * from './actions'
+
+export default from './getters'
```

Comment:
I ended up killing the index file in my implementation... the extra boilerplate just to do fancy one-line `imports` didn't seem necessary. It's pretty though.